src/components/side-panel/side-panel.styles.tsx

```diff
@@ -21,7 +23,7 @@ export const Wrapper = styled.div<IWrapperStyleProps>`
     display: flex;
     flex-direction: column;
     height: calc(100% - ${(props) => props.$offset}rem);
-    width: 36.8rem;
+    width: ${({ $narrow }) => ($narrow ? "36.8rem" : "31.2rem")};
```

Comment:
found it slightly confusing as the meaning of this variable is reversed? the element gets wider when `narrow` is true

Reply:
oops realised I named in wrongly